test/test_prototype_datasets_utils.py

```diff
@@ -48,6 +51,187 @@ def test_read_flo(tmpdir):
     torch.testing.assert_close(actual, expected)
 
 
+# This cannot be defined on the TestOnlineResource class, since it is used in a `@pytest.mark.parametrize` decorator on
+# a method on said class
+def _decompress_gz(path):
```

Comment:
We don't have a decompressing util already written somewhere that we can use for that?

Reply:
We have, but it is the same we use for `preprocess="decompress"`:

https://github.com/pytorch/vision/blob/d9a69506109cb970c7568c27cbeda1f9ffb7ad70/torchvision/datasets/utils.py#L343

https://github.com/pytorch/vision/blob/d9a69506109cb970c7568c27cbeda1f9ffb7ad70/torchvision/prototype/datasets/utils/_resource.py#L58-L60

If we are ok in re-using `_decompress`, I can remove this function.